components/Home/sidebar/sidebar_items/BountiesSidebarItem.tsx

```diff
@@ -0,0 +1,118 @@
+import { css, StyleSheet } from "aphrodite";
+import { FEDocType } from "~/config/utils/getUnifiedDocType";
+import { ID, NullableString } from "~/config/types/root_types";
+import { ReactElement } from "react";
+import AuthorFacePile from "~/components/shared/AuthorFacePile";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  bountyAmount: number;
+  bountyContentSnippet: string;
+  createdByAuthor: any;
+  documentType: FEDocType;
+  expirationDate: string;
+  isCommentBounty: boolean;
+  relatedDocID: ID;
+  slug: NullableString;
+};
+
+export default function BountiesSidebarItem({
+  bountyAmount,
+  bountyContentSnippet,
+  createdByAuthor,
+  documentType,
+  isCommentBounty,
+  relatedDocID,
+  slug,
+}: Props): ReactElement {
+  const roundedOfferAmount = bountyAmount;
+
+  return (
+    <div className={css(styles.bountiesSidebarItemContainer)}>
+      {/* NOTE: href is subject to change */}
+      <a
+        style={{ textDecoration: "none" }}
+        href={`/${
+          documentType === "question" ? "post" : documentType
+        }/${relatedDocID}/${slug ?? ""}${isCommentBounty ? "#comments" : ""}`}
+      >
+        <div className={css(styles.bountiesSidebarItem)}>
+          <div className={css(styles.bountiesSidebarItemHeader)}>
+            <AuthorFacePile
+              authorProfiles={[createdByAuthor]}
+              withAuthorName
+              imgSize={12.3}
+            />
+            <span className={css(styles.bountiesSidebarTitle)}>
+              <span>{"is offering "}</span>
+              <span className={css(styles.bountiesAmount)}>
+                {roundedOfferAmount}
+                {" RSC"}
+              </span>
+              <img
+                style={{ width: 12, marginLeft: 5 }}
+                src={"/static/icons/coin-filled.png"}
+                draggable={false}
+                alt="RSC Coin"
+              />
+            </span>
+          </div>
+          <div className={css(styles.bountiesSidebarItemContent)}>
+            {bountyContentSnippet}
+          </div>
+        </div>
+      </a>
+    </div>
+  );
+}
+
+const styles = StyleSheet.create({
+  bountiesSidebarItemContainer: {
+    borderLeft: "2px solid transparent",
+    color: colors.TEXT_GREY(1),
+    cursor: "pointer",
+    padding: "12px 0",
+    textDecoration: "none",
+    width: "100%",
+    ":hover": {
+      background: colors.LIGHT_GREY_BACKGROUND,
+      borderLeft: "2px solid blue",
```

Comment:
I think the default BLUE is probably not consistent with the RH blue

Reply:
Fixed on `rightSidebarMigrate4`